From da8d0f7800b3be78af3ed2b089fd6714b700d2eb Mon Sep 17 00:00:00 2001
From: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
Date: Thu, 10 Sep 2020 12:56:07 +0530
Subject: [PATCH 1663/1852] dt-bindings: serial: Add xilinx compatible

commit d7bda34a017d911f8bf6e0e3737473c0073ad47c from
https://github.com/Xilinx/linux-xlnx.git

Add the arm,xlnx-sbsa-uart compatible.
Xilinx versal uart is similar to sbsa-uart except that it has
termios configurable.

Signed-off-by: Shubhrajyoti Datta <shubhrajyoti.datta@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/serial/pl011.yaml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/devicetree/bindings/serial/pl011.yaml b/Documentation/devicetree/bindings/serial/pl011.yaml
index 1a64d59152aa..7df9b754e67f 100644
--- a/Documentation/devicetree/bindings/serial/pl011.yaml
+++ b/Documentation/devicetree/bindings/serial/pl011.yaml
@@ -20,6 +20,7 @@ select:
         enum:
           - arm,pl011
           - zte,zx296702-uart
+          - arm,xlnx-sbsa-uart
   required:
     - compatible
 
@@ -32,6 +33,10 @@ properties:
       - items:
           - const: zte,zx296702-uart
           - const: arm,primecell
+      - items:
+          - const: arm,xlnx-sbsa-uart
+          - const: arm,primecell
+
 
   reg:
     maxItems: 1
-- 
2.31.1

